contrastive_vi/module/total_contrastive_vi.py

```diff
@@ -686,14 +692,15 @@ def loss(
             background_losses["reconst_loss_protein"]
             + target_losses["reconst_loss_protein"]
         )
-        reconst_losses = dict(
-            reconst_loss_gene=reconst_loss_gene,
-            reconst_loss_protein=reconst_loss_protein,
+
+        wasserstein_loss = (
+            torch.norm(inference_outputs["background"]["qs_m"], dim=-1)**2
+            + torch.norm(torch.sqrt(inference_outputs["background"]["qs_v"]), dim=-1)**2
```

Comment:
Again, this can be computed more efficiently as the sum of the variance terms.

Reply:
Fixed